src/main/java/com/netgrif/application/engine/petrinet/service/PetriNetService.java

```diff
@@ -143,7 +147,7 @@ public void evictCache(PetriNet net) {
      * Get read only Petri net.
      */
     @Override
-    @Cacheable(value="petriNetCache", unless="#result == null")
+    @Cacheable(value = "petriNetCache", unless = "#result == null")
     public PetriNet get(ObjectId petriNetId) {
         Optional<PetriNet> optional = repository.findById(petriNetId.toString());
         if (!optional.isPresent()) {
```

Comment:
We can use `optional.isEmpty()` or just invert if to `if (optional.isPresent()) { return option.get() }`